src/app/beer_garden/api/http/processors.py

```diff
@@ -22,8 +20,6 @@ def put(self, event):
 def websocket_publish(item):
     """Will serialize an event and publish it to all event websocket endpoints"""
     try:
-        beer_garden.api.http.io_loop.add_callback(
-            EventSocket.publish, SchemaParser.serialize(item, to_string=True)
-        )
+        beer_garden.api.http.io_loop.add_callback(EventSocket.publish, item)
```

Comment:
Does this now receive the event after it's been serialized? In that case, the docstring should be updated.

Reply:
Docstring updated.